tests/ui-toml/toml_disallowed_method/conf_disallowed_method.stderr

```diff
@@ -11,6 +11,8 @@ error: use of a disallowed method `regex::re_unicode::Regex::is_match`
    |
 LL |     re.is_match("abc");
    |     ^^^^^^^^^^^^^^^^^^
+   |
+   = note: no matching allowed
```

Comment:
I wonder if we should somehow indicate that this is from the configuration? But I can't think of a good way to do it.

Reply:
maybe something like `note: from clippy.toml: no matching allowed`? or (and this would be harder) putting a span pointing to the line of the clippy.toml file?

Reply:
how about `note: no matching allowed (from clippy.toml)`